src/main/java/org/opentripplanner/middleware/triptracker/ManageTripTracking.java

```diff
@@ -52,7 +55,8 @@ private static TrackingResponse doUpdateTracking(Request request, TripTrackingDa
 
```

Comment:
Bad merge at line 40: should be `return doUpdateTracking(request, tripData, create);`